### PR TITLE
Add routegroup and fabricgateway count metrics

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: kubernetes/adapter.go [hunk 1]
```diff
@@ -26,27 +26,14 @@ type Adapter struct {
...
-type ingressType int
+type IngressType string
 
 const (
-	ingressTypeIngress ingressType = iota + 1
-	ingressTypeRouteGroup
-	ingressTypeFabric
+	TypeIngress    IngressType = "ingress"
+	TypeRouteGroup IngressType = "routegroup"
+	TypeFabric     IngressType = "fabric"
```
Comment (AlexanderYastrebov) [inline kubernetes/adapter.go:34]: maybe should name this `TypeFabricGateway IngressType = "fabricgateway"` for consistency

Comment (szuecs) [reply kubernetes/adapter.go:34]: yes I think TypeFabricGateway makes sense

**Comment by szuecs** [comment]
lgtm

File: kubernetes/adapter.go (hunk 1 above)
Comment (AlexanderYastrebov) [reply kubernetes/adapter.go]: Renamed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**